R/development.R

```diff
@@ -1,4 +1,9 @@
 meta_development <- function(meta, version, bs_version = 3) {
+  # Need to re-bind locales here for translation
```

Comment:
Please remove this and instead get the development version of pkgload which takes care of this automatically.